### PR TITLE
G2 2023 v1 issue#13015

### PR DESCRIPTION
**Pull request by c21axepe**
Made changes as requested from reviewers

### CODE REVIEW

File: backend-models/sectionedservice.svg
Comment (a21jeaha) [inline backend-models/sectionedservice.svg:1]: the discuses change was rectified, which is good.
though i found a small issue regarding the class name, shouldn't it be the services name ?

Comment (c21axepe) [reply backend-models/sectionedservice.svg:1]: Changed classname as requested!

Comment (a21jeaha) [reply backend-models/sectionedservice.svg:1]: neat, great work.
It's now ready for merge.

**Review by a21jeaha — APPROVED** [review]
ready for merge

**Comment by a19hakja** [comment]
merging

**Pull request merged**